app/src/main/res/values/attrs.xml

```diff
@@ -168,7 +168,9 @@
     </declare-styleable>
 
     <declare-styleable name="InformationBlockView">
-        <attr name="text" />
+        <attr name="title" />
+        <attr name="description" />
+        <attr name="button" format="string" />
```

Comment:
You could rename the property as `buttonLabel` here as well while we're at it, to help a bit with coherence